components/dashboard/Task.js

```diff
@@ -114,19 +117,30 @@ export default function Task(props) {
   }
 
   return (
-    <Card className={clsx(classes.card, highlightLabel && classes.highlight)} data-intercom="task">
+    <Card
+      className={clsx(classes.card, highlightLabel && classes.highlight)}
+      data-intercom="task"
+      elevation={0}
+    >
       {!!highlightLabel && <div className={classes.highlightLabel}>{highlightLabel}</div>}
-
-      <CardHeader
-        title={<strong>{task.fields.Title}</strong>}
-        titleTypographyProps={{ component: 'h1', variant: 'h3' }}
-      />
+      <CardHeader title={task.fields.Title} titleTypographyProps={{ component: 'h5' }} />
```

Comment:
Let's do `{ component: 'h1', variant: 'h5' }` to preserve semantics (since the smaller headings in the card are `h3`s).